org.eclipse.buildship.oomph/model/GradleImport-1.0.ecore

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
❌ Why do we need two ecore files?

Reply:
The `GradleImport.ecore` is the master where changes are made. `GradleImport-1.0.ecore` is an output file generated by the genmodel step and is used as the schema (and pointed to from the schemaLocation). I guess this separation allows for increasing the version on incompatible model changes and still supporting older versions.

Reply:
Ah, so the 1.0.ecore is generated! It's been a while since I've worked with EMF.